src/app/Page.js

```diff
@@ -28,3 +62,54 @@ export default class Page extends React.Component<{|+children: Node|}> {
     );
   }
 }
+
+const style = StyleSheet.create({
+  header: {
+    color: "#1297A1",
+    fontFamily: '"Roboto Condensed", sans-serif',
+  },
+  body: {
+    padding: "0 100px",
+    flex: 3,
+  },
+  nav: {
+    height: 60,
+    padding: "20px 100px",
+  },
+  navItem: {
+    display: "inline-block",
+  },
+  navList: {
+    listStyle: "none",
+    paddingLeft: 0,
+    margin: 0,
+    display: "flex",
+  },
+  navLink: {
+    color: "#00848B",
+    fontFamily: "Roboto Condensed",
+    fontSize: 18,
+    textDecoration: "none",
+    ":hover": {
+      color: "#08A27F",
+      textDecoration: "underline",
+    },
+  },
+  navItemLeft: {
+    flex: 1,
+  },
+  navItemRight: {
+    marginLeft: 20,
+  },
+  logoImg: {
+    height: 20,
+  },
+  navLogo: {
+    height: 20,
+    width: 20,
+    fill: "#00848B",
```

Comment:
Please consider omitting these color declarations and instead
constructing your SVGs with `<path fill="currentColor" d={…} />`. One
advantage of this is that you only specify the colors in one location.
More importantly, this change would fix a bug in the current
implementation, wherein the `a` can be hovered even if the `svg` is not
(the `a` descends 4px below the baseline due to line-height), which
results in inconsistent styling: the cursor is `pointer` but the logo
does not appear hovered.

Reply:
I've changed where the fill is getting applied from to fix this- I wanted to keep fill out of the component itself to enable the hover color change.